### PR TITLE
Add uniqueness and bank transfer tests.

### PR DESCRIPTION
**Pull request by knz**
Some more testing, inspired by recent conversations on Gitter and the sources of other Jepsen tests.

### CODE REVIEW

File: cockroachdb/README.rst [hunk 1]
```diff
@@ -37,6 +42,65 @@ As of Jan 26 2016, following the test procedure outlined below and
...
+One table contains a single row.
+
+Jepsen sends concurrently to different nodes either read, write or
+atomic compare-and-swap operations.
+
+Concurrently, a nemesis partitions the network between the nodes randomly.
+
+Each node may report ok, the operation is known to have succeeded;
+fail, the operation is known to have failed; and unknown otherwise
+(e.g. the onnection dropped before the answer could be read).
```
Comment (bdarnell) [inline cockroachdb/README.rst]: s/onnection/connection/

File: cockroachdb/README.rst [hunk 2]
```diff
@@ -37,6 +42,65 @@ As of Jan 26 2016, following the test procedure outlined below and
...
+-----------------------------
+
+Jepsen sends appends of unique values via different
+nodes over time. 
+
+Concurrently, a nemesis partitions the network between the nodes randomly.
+
+Each node may report ok, the operation is known to have succeeded;
+fail, the operation is known to have failed; and unknown otherwise
+(e.g. the onnection dropped before the answer could be read).
```
Comment (bdarnell) [inline cockroachdb/README.rst]: Also here and in the third test.

**Comment by bdarnell** [comment]
LGTM. I assume we're currently passing the two new tests, and you're waiting on subquery support to add the negative-balance checks to the bank test?

**Comment by knz** [comment]
Yes correct.

**Pull request merged**